package.json

```diff
@@ -47,6 +47,7 @@
     "grunt-banner": "^0.3.1",
     "grunt-browserify": "3.5.1",
     "grunt-cli": "~0.1.0",
+    "grunt-concurrent": "^1.0.0",
```

Comment:
not necessary for this PR, but we probably should look through our devDeps and make sure we're still using them all and if not, get rid of them.